repos/fdbt-site/src/pages/api/salesConfirmation.ts

```diff
@@ -54,6 +54,11 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
             userDataJson = getSingleTicketJson(req, res);
         } else if (fareType === 'return') {
             userDataJson = getReturnTicketJson(req, res);
+        } else if (fareType === 'capped') {
+            if (!ticketType || ticketType === 'hybrid' || ticketType === 'pointToPointPeriod') {
+                throw new Error('Capped ticket required a type of ticket representation of geoZone or multiService.');
+            }
+            userDataJson = await getCappedTicketJson(req, res, ticketType);
```

Comment:
My linter shows Unsafe call of an `any` typed value.

Reply:
Hmm, the repo's linting doesnt because it passes on the pipeline. Could you see which function it thinks is returning an any?